pootle/templates/editor/units/xhr_tm.html

```diff
@@ -9,7 +9,9 @@
           dir="<%= store.source_dir %>"><%= PTL.editor.doDiff(unit.source[0], sugg.source) %></div>
         <div class="suggestion-translation" lang="<%= store.target_lang %>"
           dir="<%= store.target_dir %>"><%= PTL.utils.fancyHl(sugg.target) %></div>
-        <% if (sugg.project) { %>
+        <% if (!sugg.is_local) { %>
```

Comment:
Changes in this template are now gone since we ask to provide a project name when importing the translations.